README.md

```diff
@@ -59,7 +57,7 @@ const myRule = {
 
   // Takes the data returned from `tokenize` and returns an array of rolls
   // this is so we can see what every individual dice roll was if we want
-  roll: ({ numDice }) => {
+  roll: ({ numDice }, random) => {
```

Comment:
Part of me thinks we should have this new argument be an object, to future-proof it if we ever want to add more helpers later on.

```suggestion
  roll: ({ numDice }, { random }) => {
```

We can always just make this change later if we need it though.

Reply:
Yes, I agree! I can send a follow up pr!